scaffold/acquia/settings.acquia.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+/**
+ * @file
+ * Setup Acquia Cloud settings.
+ */
```

Comment:
Wouldn't this file already exist in the repo?

Reply:
@deviantintegral only if you've included a particular composer dependency that also brings this in (in a very roundabout way)